src/CONST.js

```diff
@@ -991,7 +991,9 @@ const CONST = {
         CODE_2FA: /^\d{6}$/,
         ATTACHMENT_ID: /chat-attachments\/(\d+)/,
         HAS_COLON_ONLY_AT_THE_BEGINNING: /^:[^:]+$/,
-        NEW_LINE_OR_WHITE_SPACE: /[\n\s]/g,
+
+        // eslint-disable-next-line no-misleading-character-class
+        NEW_LINE_OR_WHITE_SPACE_OR_EMOJI: /[\n\s\p{Extended_Pictographic}\u200d\u{1f1e6}-\u{1f1ff}\u{1f3fb}-\u{1f3ff}\u{e0020}-\u{e007f}\u20E3\uFE0F]|[#*0-9]\uFE0F?\u20E3/gu,
```

Comment:
Can't we substitute the variable instead of copy-paste two to avoid repeating Regex twice?

Reply:
@Santhosh-Sellavel To resolve this problem, we can create new function to concat regex like [this](https://masteringjs.io/tutorials/fundamentals/concat-regexp#:~:text=To%20concatenate%20a%20regular%20expression%20in%20JavaScript%2C%20you%20can%20use,(options%20for%20the%20RegExp).&text=You%20are%20responsible%20for%20removing%20duplicate%20flags.) or we can split multiple time like [this](https://stackoverflow.com/a/36976348)

Reply:
Cool, keep us updated

Reply:
@Santhosh-Sellavel I just updated, help to review